spec/lib/rmt/cli/systems_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'rails_helper'
 require_relative '../../../../engines/registration_sharing/lib/registration_sharing'
 
+# rubocop:disable RSpec/NestedGroups
```

Comment:
Is really there the need to disable `rubocop` here? Wouldn't it be possible to write a less deep context?